cli/pcluster/config/pcluster_config.py

```diff
@@ -358,8 +359,12 @@ def __check_account_capacity(self):  # noqa: C901
         # Check for insufficient Account capacity
         compute_subnet = vpc_section.get_param_value("compute_subnet_id")
         master_subnet = vpc_section.get_param_value("master_subnet_id")
+        vpc_security_group = vpc_section.get_param_value("vpc_security_group_id")
         if not compute_subnet:
             compute_subnet = master_subnet
+        security_groups_ids = []
+        if vpc_security_group:
+            security_groups_ids.append(vpc_security_group)
```

Comment:
Do we know for sure that this security group exists before this is run?

Reply:
Ah yeah, the security group validator would have already been run. Nevermind.

Reply:
Good question. I tried with an invalid security group and it is the error:
```
$ pcluster create test -t test
Beginning cluster creation for cluster: test
ERROR: The configuration parameter 'vpc_security_group_id' generated the following errors:
Invalid id: "sg-xxx"
```